Objects/Converters/ConverterRevit/ConverterRevitTests/ConverterRevitTestsShared/SpeckleUtils.cs

```diff
@@ -46,6 +47,7 @@ internal static class SpeckleUtils
       {
         tcs.TrySetException(exception);
       }
+#pragma warning restore CA1031 // Do not catch general exception types
```

Comment:
Not sure about this one, possible we consider it a TL handler because of what it is? Not convinced though

Reply:
Assuming we ensure the task completion source is always observed. I think its safe to catch all here.
However, to be extra safe. In the few places of Core where we are using TaskCompletionSources, I'm using this pattern 

https://github.com/specklesystems/speckle-sharp/blob/d0952efb1965bbd75e64ddebd828e86c368b2290/Core/Core/Serialisation/DeserializationWorkerThreads.cs#L51-L63C6

 we may consider doing something similar here.

Reply:
That being said.. I'm a bit puzzled why we're using a Task completion source here anyway. since we await the APIContext call anyway...